mongodb/mongo-init.js

```diff
@@ -0,0 +1,19 @@
+db = db.getSiblingDB("admin") 
+db.auth("admin","admin") 
```

Comment:
Some security issues come to mind looking at this, we should look into best practices for passing credentials in without hard coding them.

Reply:
I agree. I will look into more secure ways of managing our credentials